assemblies/assembly-enabling-authentication.adoc

```diff
@@ -1,39 +1,46 @@
 [id='enabling-authentication']
 = Enabling authentication in {product}
 
-Authentication within {product} facilitates user sign-in, identification, and access to external resources.
-It supports multiple authentication providers.
-
-Authentication providers are typically used in the following ways:
-
-- One provider for sign-in and identification.
-- Additional providers for accessing external resources.
-
-The {product} supports the following authentication providers:
-
-//Auth0:: `auth0`
-//Atlassian:: `atlassian`
-Microsoft Azure::
-`microsoft`
-//Microsoft Azure Easy Auth:: `azure-easyauth`
-//Bitbucket:: `bitbucket`
-//Bitbucket Server:: `bitbucketServer`
-//Cloudflare Access:: `cfaccess`
-GitHub:: `github`
-//GitLab:: `gitlab`
-//Google:: `google`
-//Google IAP:: `gcp-iap`
-Keycloak:: `oidc`
-//Okta:: `okta`
-//OAuth 2 Custom Proxy:: `oauth2Proxy`
-//OneLogin:: `onelogin`
-//SAML:: `saml`
-
-For each provider that you want to use, follow the dedicated procedure to complete the following tasks:
+Your organization certainly requires you to identify and authorize users to give them access to resources, such as {product}.
+
+In {product-short}, authentication and authorization are two separate processes:
+
+. Authentication is defining the user identity, and passing on this information to {product-short}.
+Read following chapters to progressively configure authentication in {product-short}.
+
+. Authorization is defining what actions an authenticated identity can perform in {product-short}.
+See link:{authorization-book-url}[{authorization-book-title}].
+
+To start exploring {product-short} features, do not worry about authentication and authorization: after the default installation, you can authenticate with the Guest user, and use all the application features.
```

Comment:
```suggestion
[NOTE]
====
To explore {product-short} features, you do not need to configure authentication and authorization. By default, {product-short} allows you to authenticate with the Guest user which enables you to access authorized features associated with the guest user role.
====
```
I'd suggest rephrasing this content and perhaps making it a note. I wouldn't tell the user to not worry about authorization and authorization. Also, I'm not sure what features the Guest user role allows but it's not "all application features" - maybe a SME has a better definition of what the Guest user role can access?

Reply:
This is not a note, this is the first step in the user journey.